### PR TITLE
Adding container profile overview

### PR DESCRIPTION
**Pull request by matthiaz**
Adding an overview of the amount of vCPU and memory each container profile gets.

## Why

It is very hard for customers to figure out which profile size will get them the correct vCPU and memory they want. 
They can get the vCPU and memory from the container, but that requires numerous deployments. It's much easier to have this documented. 

## What's changed

Added a section on what the container profiles actually give you in terms of vCPU and memory

### CODE REVIEW

File: sites/platform/src/create-apps/app-reference.md [hunk 1]
```diff
@@ -175,6 +175,51 @@ To do so, set `size` to one of the following values:
...
+
+We use different container profiles to ensure that each container gets the right amount of vCPU and memory for its specific use case. Ideally you want to give databases the biggest part of your memory, and apps the biggest part of the vCPU.
+
+There are 3 types of container profiles. 
+
+1. HIGH_CPU: php, nodejs, python, ... 
+2. HIGH_MEMORY: mariadb, mongodb, mysql-oracle, ...
+3. BALANCED: redis, ...
+
+The size of the container + the container profile determines how much vCPU and memory it gets.
```
Comment (trolologuy) [inline sites/platform/src/create-apps/app-reference.md:188]:
```suggestion
The size of the container and the container profile determines how much vCPU and memory (in [MB](https://docs.platform.sh/glossary.html#mb)) it gets.
```

File: sites/platform/src/create-apps/app-reference.md [hunk 2]
```diff
@@ -175,6 +175,51 @@ To do so, set `size` to one of the following values:
 
 The total resources allocated across all apps and services can't exceed what's in your plan.
 
+### Container profiles: vCPU and Memory
+
+We use different container profiles to ensure that each container gets the right amount of vCPU and memory for its specific use case. Ideally you want to give databases the biggest part of your memory, and apps the biggest part of the vCPU.
+
+There are 3 types of container profiles. 
+
+1. HIGH_CPU: php, nodejs, python, ...
```
Comment (trolologuy) [inline sites/platform/src/create-apps/app-reference.md:184]:
```suggestion
1. `HIGH_CPU`: php, nodejs, python, ... 
```

File: sites/platform/src/create-apps/app-reference.md [hunk 3]
```diff
@@ -175,6 +175,51 @@ To do so, set `size` to one of the following values:
...
 
+### Container profiles: vCPU and Memory
+
+We use different container profiles to ensure that each container gets the right amount of vCPU and memory for its specific use case. Ideally you want to give databases the biggest part of your memory, and apps the biggest part of the vCPU.
+
+There are 3 types of container profiles. 
+
+1. HIGH_CPU: php, nodejs, python, ... 
+2. HIGH_MEMORY: mariadb, mongodb, mysql-oracle, ...
+3. BALANCED: redis, ...
```
Comment (trolologuy) [inline sites/platform/src/create-apps/app-reference.md:186]:
```suggestion
3. `BALANCED`: redis, ...
```

File: sites/platform/src/create-apps/app-reference.md [hunk 4]
```diff
@@ -175,6 +175,51 @@ To do so, set `size` to one of the following values:
...
+
+The size of the container + the container profile determines how much vCPU and memory it gets.
+
+1. HIGH_CPU
```
Comment (trolologuy) [inline sites/platform/src/create-apps/app-reference.md:190]:
```suggestion
#### HIGH_CPU 
```

File: sites/platform/src/create-apps/app-reference.md [hunk 5]
```diff
@@ -175,6 +175,51 @@ To do so, set `size` to one of the following values:
...
+| Size |  CPU  | MEMORY  |
+| ---- | ----- | ------- |
+| S    | 0.40  | 128MB   |
+| M    | 0.40  | 128MB   |
+| L    | 1.20  | 256MB   |
+| XL   | 2.50  | 384MB   |
+| 2XL  | 5.00  | 768MB   |
+| 4XL  | 10.00 | 1536MB  |
+
+2. HIGH_MEMORY
```
Comment (trolologuy) [inline sites/platform/src/create-apps/app-reference.md:201]:
```suggestion
#### HIGH_MEMORY 
```

File: sites/platform/src/create-apps/app-reference.md [hunk 6]
```diff
@@ -175,6 +175,51 @@ To do so, set `size` to one of the following values:
...
+| Size | CPU  | MEMORY   |
+| ---- | ---- | -------- |
+| S    | 0.25 | 128MB    |
+| M    | 0.25 | 288MB    |
+| L    | 0.40 | 1280MB   |
+| XL   | 0.75 | 2624MB   |
+| 2XL  | 1.50 | 5248MB   |
+| 4XL  | 3.00 | 10496MB  |
+
+3. BALANCED
```
Comment (trolologuy) [inline sites/platform/src/create-apps/app-reference.md:212]:
```suggestion
#### BALANCED 
```

File: sites/platform/src/create-apps/app-reference.md [hunk 7]
```diff
@@ -175,6 +175,51 @@ To do so, set `size` to one of the following values:
...
+2. HIGH_MEMORY: mariadb, mongodb, mysql-oracle, ...
+3. BALANCED: redis, ...
+
+The size of the container + the container profile determines how much vCPU and memory it gets.
+
+1. HIGH_CPU 
+
+| Size |  CPU  | MEMORY  |
+| ---- | ----- | ------- |
+| S    | 0.40  | 128MB   |
```
Comment (trolologuy) [inline sites/platform/src/create-apps/app-reference.md:194]:
```suggestion
| S    | 0.40  | 128 MB  |
```
Adapt accordingly for all lines

File: sites/platform/src/create-apps/app-reference.md [hunk 8]
```diff
@@ -175,6 +175,51 @@ To do so, set `size` to one of the following values:
 
 The total resources allocated across all apps and services can't exceed what's in your plan.
 
+### Container profiles: vCPU and Memory
+
+We use different container profiles to ensure that each container gets the right amount of vCPU and memory for its specific use case. Ideally you want to give databases the biggest part of your memory, and apps the biggest part of the vCPU.
+
+There are 3 types of container profiles.
```
[the remark this answers is not in the document]
Comment (matthiaz) [reply sites/platform/src/create-apps/app-reference.md:182]: There are 3 container profiles on PlatformSh. Yes there are more for upsun, but there aren't any more on psh, right?

**Comment by AnouckColson** [comment]
Addressed in #3929

**Pull request closed without merging**